.editorconfig

```diff
@@ -1,7 +1,7 @@
 root = true
 
 [*]
-max_line_length = 100
+max_line_length = 120
```

Comment:
Super nitpicky, but any real reason why line length is 120 for this repo but 100 in neovim? Not really biased towards any value, just that the discrepancy raised eyebrows.

Reply:
No reason except that's what this repo defined for .stylua so I just copied it here. Changing it might involve some churn